client/lib/abtest/active-tests.js

```diff
@@ -111,4 +111,12 @@ export default {
 		},
 		defaultVariation: 'public',
 	},
+	crowdsignalNameBasedSignup: {
+		datestamp: '20181114',
```

Comment:
Will this be the actual date it starts?

Reply:
Yes. I'm going to set this to Monday 19 November for now in case we need to take any additional events into consideration.